docs/source/certs_management.md

```diff
@@ -379,3 +379,77 @@ Certificate:
          9c:02:20:51:33:42:5e:a0:8a:2a:ec:f5:83:46:f0:99:6a:7e:
          eb:a8:97:1f:30:99:9d:ae:8d:ef:36:07:da:bb:67:ed:80
 ```
+
+## Certificate Renewal
+
+Enrollment certificates and TLS certificates for all nodes and clients should be updated before they expire in order to avoid disruption of service.
+Obtain updated certificates from the issuing Fabric Certificate Authority (CA) by using the Fabric CA enroll or reenroll function.
+The reenroll function allows the reuse of an existing private key, which is especially important for orderer TLS certificates.
+
+### Certificate Renewal of Expired Certificates
+
+If certificates expire before reenrollment you must enable the `reenrollignorecertexpiry` option on the Fabric CA server prior to reenrollment.
```

Comment:
So this says user can "reenroll" either before or after expiration. Confirm that's true. Otherwise rephrase to make clearer.

Reply:
It's true, clarified!